doc/BLOG.md

```diff
@@ -0,0 +1,166 @@
+```
+$ pgbackrest --stanza=demo --type=incr --repo=1 backup
+
+<...>
+INFO: backup command end: completed successfully (12525ms)
+```
+
+Here we can see that the incremental backup is nearly as large as the full backup, 52.8MB vs 55.5MB. This is expected since the bulk of the database is contained in a single file and by default incremental backups copy the entire file if any part of the file has changed.
+```
+$ pgbackrest --stanza=demo --repo=1 info
+
+full backup: 20230520-082323F
+    database size: 995.7MB, database backup size: 995.7MB
+    repo1: backup size: 55.5MB
+
+incr backup: 20230520-082323F_20230520-082934I
+    database size: 995.7MB, database backup size: 972.8MB
+    repo1: backup size: 52.8MB
+```
+
+However, on repo2 with block incremental enabled, the backup is significantly faster.
+```
+$ pgbackrest --stanza=demo --type=incr --repo=2 backup
+
+<...>
+INFO: backup command end: completed successfully (3589ms)
+```
+
+And also much smaller, 943KB vs 52.8MB on the repo without block incremental enabled. This is more than 50x improvement in backup size! Note that the block incremental backup feature also works with differential backups.
+```
+$ pgbackrest --stanza=demo --repo=2 info
+
+full backup: 20230520-082438F
+    database size: 995.7MB, database backup size: 995.7MB
+    repo2: backup size: 56MB
+
+incr backup: 20230520-082438F_20230520-083027I
+    database size: 995.7MB, database backup size: 972.8MB
+    repo2: backup size: 943.3KB
+```
+
+Block incremental also improves the efficiency of the delta restore command. Here we stop the cluster and perform a delta restore back to the full backup in repo 1:
+```
+$ pg_ctlcluster 12 demo stop
+$ pgbackrest --stanza=demo --delta --repo=1 --set=20230526-053458F restore
+
+<...>
+INFO: restore command end: completed successfully (3697ms)
+```
+As we saw above the main table is contained in a single file, so the restore must copy and decompress the entire file from repo 1 (compressed size 30.4MB) because it was changed since the full backup.
+
+To test the full backup restore in repo 2 we need to first restore the cluster to the most recent backup in repo 2:
+```
+pgbackrest --stanza=demo --delta --repo=2 restore
+```
+
+And then perform a delta restore back to the full backup in repo 2:
+```
+pgbackrest --stanza=demo --delta --repo=2 --set=20230526-053406F restore
+
+<...>
+INFO: restore command end: completed successfully (1536ms)
+```
+This is noticeably faster even on our fairly small demo database. When storage latency is high (e.g. S3) the performance improvement will be more pronounced. With block incremental enabled delta restore only had to copy 3.5MB of the main table file from repo 2, as compared to 30.4MB from repo 1.
```

Comment:
```suggestion
This is noticeably faster even on our fairly small demo database. When storage latency is high (e.g. S3) the performance improvement will be more pronounced. With block incremental enabled, delta restore only had to copy 3.5MB of the main table file from repo 2, as compared to 30.4MB from repo 1.
```

Reply:
I was on the fence about the comma, so this works for me.